docs/source/pipeline.md

```diff
@@ -30,22 +46,21 @@ groupby_features = [
 )
 ```
 
-#### Outputs
+### Outputs
 
-NVTabular outputs parquet files with the preprocessed data. The parquet files can be (Hive) partitioned by a categorical column (e.g. day, company), as in the following example.
+NVTabular outputs parquet files with the preprocessed data. The parquet files can be (Hive) partitioned by a categorical column, such as day and company, as shown in the following example:
```

Comment:
I believe that Parquet is it's own project and we should initial cap it and treat it like a proper noun.

The second item is that NVT _can_ save data in the Parquet format: "NVTabular can save preprocessed data in Parquet files. The files can be..."  I don't know why Hive is in the sentence or in parens.  Consider removing it.